src/lib.rs

```diff
@@ -1,6 +1,8 @@
 #![warn(missing_docs)]
 #![doc = include_str!("../README.md")]
 
+#[cfg(feature = "internal_bevy_egui")]
```

Comment:
Nit: May want to opt for something like 
```rust
#[cfg(any(feature = "bevy_egui_0_37", feature = "bevy_egui_0_38"))]
```

Reply:
Thanks for the suggestion. That list might get long if there are a lot of bevy_egui releases. Keeping it for now. If it turns out there are only one or two per bevy release, I'll change it.